src/client/components/Button/Button.styles.css

```diff
@@ -1,11 +1,12 @@
 .button {
-    color: white;
-    border: none;
-    padding: 15px 32px;
-    text-align: center;
-    text-decoration: none;
-    font-size: 16px;
-    border-radius: 4px;
-    cursor: pointer;
-    display: inline-block;
-}
\ No newline at end of file
+  color: white;
```

Comment:
Hm, is there any reason why we need to change the original styles?

Reply:
Nothing.  I just choose  the default